src/main/resources/db/migration/V2021.12.13.1814__addOttEnglishName.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `otts` ADD `english_name` VARCHAR NOT NULL;
```

Comment:
```suggestion
ALTER TABLE `otts` ADD `english_name` VARCHAR(255) NOT NULL;
```

h2에서는 varchar 크기 안줘도 돌아가는데 mysql에서는 오류날거에요 🤣